package/network/services/hostapd/src/src/utils/build_features.h

```diff
@@ -7,10 +7,6 @@ static inline int has_feature(const char *feat)
 	if (!strcmp(feat, "eap"))
 		return 1;
 #endif
-#ifdef CONFIG_IEEE80211N
-	if (!strcmp(feat, "11n"))
```

Comment:
Is LuCi not checking this any more?

Reply:
Nope, not for any mode. I added ac/ax here: https://github.com/openwrt/luci/pull/6104
But even if something would, it would indicate 11n isn't supported, which this PR doesn't change. Because `#ifdef CONFIG_IEEE80211N` doesn't hit since it doesn't exists :)